en/flight_controller/airlink.md

```diff
@@ -0,0 +1,1003 @@
+   </td>
+   </tr>
+   <tr>
+   <td>9</td>
+   <td>BUZZER</td>
+   <td>OUT</td>
+   <td>+5V</td>
+   <td>Buzzer output</td>
+   </td>
+   </tr>
+   <tr>
+   <td>10</td>
+   <td>GND</td>
+   <td></td>
+   <td></td>
+   <td>Ground</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+- **CAN1 - JST GH SM04B-GHS-TB**
+
+   <table>
+   <tbody>
+   <tr>
+   <th>Pin number </th>
+   <th>Pin name </th>
+   <th>Direction </th>
+   <th>Voltage </th>
+   <th>Function </th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>5V</td>
+   <td>OUT</td>
+   <td>+5V</td>
+   <td>Power supply output</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>CAN1_H</td>
+   <td>I/O</td>
+   <td>+5V</td>
+   <td>CAN 1 High (120Ω)</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN1_L</td>
+   <td>I/O</td>
+   <td>+5V</td>
+   <td>CAN 1 Low (120Ω)</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td></td>
+   <td></td>
+   <td>Ground</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+- **CAN2 - JST GH SM04B-GHS-TB**
+
+   <table>
+   <tbody>
+   <tr>
+   <th>Pin number </th>
+   <th>Pin name </th>
+   <th>Direction </th>
+   <th>Voltage </th>
+   <th>Function </th>
+   </tr>
+   <tr>
+   <td>1</td>
+   <td>5V</td>
+   <td>OUT</td>
+   <td>+5V</td>
+   <td>Power supply output</td>
+   </tr>
+   <tr>
+   <td>2</td>
+   <td>CAN2_H</td>
+   <td>I/O</td>
+   <td>+5V</td>
+   <td>CAN 2 High (120Ω)</td>
+   </tr>
+   <tr>
+   <td>3</td>
+   <td>CAN2_L</td>
+   <td>I/O</td>
+   <td>+5V</td>
+   <td>CAN 2 Low (120Ω)</td>
+   </tr>
+   <tr>
+   <td>4</td>
+   <td>GND</td>
+   <td></td>
+   <td></td>
+   <td>Ground</td>
+   </tr>
+   </tbody>
+   </table>
+
+- **CAMERA - FPC 30 pin, 0.5mm pitch**
+
+
+### Rear side
+
+![Back side](../../assets/flight_controller/airlink/airlink-interfaces-back.jpg)
+
+- **Rear side interfaces:**
+
+   - SBUS input
+   - 16 PWM output channels
+   - 2x LTE antenna sockets (MIMO)
+   - WiFi antenna socket (AP & Station modes)
+
+
+# Serial mapping
+
+AIRLink has a large number of internal and external serial ports:
+
+   <table>
+   <tbody>
+   <tr>
+   <th>Serial </th>
+   <th>UART </th>
+   <th>Function </th>
+   </tr>
+   <tr>
+   <td>Serial 0</td>
+   <td>USB</td>
+   <td>Console</td>
+   </tr>
+   <tr>
+   <td>Serial 1</td>
+   <td>UART 7</td>
+   <td>Telemetry 1</td>
+   </tr>
+   <tr>
+   <td>Serial 2</td>
+   <td>UART 5</td>
+   <td>Telemetry 2 (used internally with Mission Computer)</td>
+   </tr>
+   <tr>
+   <td>Serial 3</td>
+   <td>USART 1</td>
+   <td>GPS 1</td>
+   </tr>
+   <tr>
+   <td>Serial 4</td>
+   <td>UART 8</td>
+   <td>GPS 2</td>
+   </tr>
+   <tr>
+   <td>Serial 5</td>
+   <td>USART 3</td>
+   <td>Debug console (internal connector)</td>
+   </tr>
+   <tr>
+   <td>Serial 6</td>
+   <td>USART 2</td>
+   <td>Telemetry 3</td>
+   </tr>
+   <tr>
+   <td>Serial 7</td>
+   <td>UART 4</td>
+   <td>External UART</td>
+   </tr>
+   </tbody>
+   </table>
+
+
+
+## RC Input
+
+![RC Input](../../assets/flight_controller/airlink/airlink-rc-input.jpg)
```

Comment:
Image below text where possible.

Reply:
Done here and in a few other places where it was possible.